Tests/Functions/Public/Get-Blank.Tests.ps1

```diff
@@ -3,6 +3,7 @@
 Describe 'Get-Blank' {
 
     It 'should not produce output' {
+        $breakvar = $true;
```

Comment:
This doesn't seem to be used. Is it needed anywhere? 🙂

Reply:
Yup, thought I removed that. It's what I use for setting breakpoints. I'll get that removed.